libmscore/textbase.h

```diff
@@ -243,11 +243,17 @@ class TextBase : public Element {
       int  hexState                 { -1    };
       bool _primed                  { 0 };
 
+      QRectF _basebbox; // used to rotate and offset the text
+      int _offsetX { 0 }, _offsetY { 0 };
```

Comment:
Offsets are best replaced with a single variable of type QPoint

Reply:
That's great, I'll take care of it later today :-)

Reply:
@igorkorsukov I used QPointF instead of QPoint because e.readPoint() returns a QPointF.

I wanted to hear your opinion on the following matter. Should I change the arguments of setOffsets/setInstrumentNameOffset from 2 ints(x and y) to a QPointF? I like how it is very simple to just pass 2 numbers and be done with it, but I can't help but think that passing a QPointF is cleaner.